src/ReactiveSocket.cpp

```diff
@@ -192,6 +193,8 @@ void ReactiveSocket::metadataPush(std::unique_ptr<folly::IOBuf> metadata) {
 }
 
 bool ReactiveSocket::createResponder(
+    const std::shared_ptr<RequestHandlerBase>& handler,
```

Comment:
Looks like this method doesn't actually retain `handler` in any way, so it seems like it could just be a `const RequestHandlerBase&`.